shared/chat/conversation/attachment-popup/container.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 import * as Types from '../../../constants/types/chat'
 import * as ChatGen from '../../../actions/chat-gen'
-import * as KBFSGen from '../../../actions/kbfs-gen'
```

Comment:
why change this the existing code for this? why not just leave it

Reply:
To start migrating to the new stuff.

Reply:
we should chat